README.md

```diff
@@ -241,3 +242,15 @@ ib-sriov supports the following [CNI's Capabilities / Runtime Configuration](htt
 
 EOF
 ```
+# SR-IOV Network Operator
+[SR-IOV Network Operator](https://github.com/openshift/sriov-network-operator) can manages the SR-IOV interfaces on the nodes e.g. change the number of VFs on the node, it can be used also to change the link type for the interfaces ETH to IB and vice versa, the [network policy example](https://github.com/openshift/sriov-network-operator/blob/master/deploy/crds/sriovnetwork.openshift.io_v1_sriovnetworknodepolicy_cr.yaml#L38) shows how to use the operator to change the link type for the given PCI Addresses of PFs.
+
+# System Requirements and Topology
+InfiniBand SR-IOV CNI works with kernel 5.5 which supports rdma namespace isolation and read/write GUID of the VFs.
+The network topology for the InfiniBand SR-IOV CNI it is expected to has a node on the same network with OpenSM, the following diagram captures the InfiniBand SR-IOV topology for a two-node k8s cluster with external OpenSM node.
```

Comment:
I don't understand this sentence. What are you trying to convey here ?

Reply:
It is a description for the network topology, trying to explain the connections in the image

Reply:
I would reword this sentence then.

The recommended network topology for a Kubernetes deployment with Infiniband as a secondary network is as follows:
- Two physical networks, one Ethernet network used as Kubernetes management and Pod primary network (these can be separate) and another Infiniband network interconnecting Kubernetes worker nodes
- Worker nodes where ib-sriov CNI is invoked are expected to have connectivity through the infiniband fabric with the subnet manager (SM), either running on a managed Infiniband Switch or another node.. (i.e there needs to be an active SM in the fabric)